specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1131,12 +1131,12 @@
       "properties": {
         "id": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
May I ask why this property is writeable? Per my understanding, service side is able to get the vm info via the parameter [VmNameParameter ](https://github.com/Azure/azure-rest-api-specs/blob/4bab0bfdf7b1e2e3a50eeda420c2891d087d1d85/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L66)while creating this resource, right?